common/alter_operation_error.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: default
+parent: Common
+---
+
+# Add Operation Error
```

Comment:
```markdown
# Alter Operation Error
```

Reply:
There were a lot of concerns here. I thought I had revised this file and apparently missed some sections. This has been remedied.